content/200-concepts/100-components/02-prisma-client/260-client-extensions/100-client-extensions-model.mdx

```diff
@@ -72,11 +72,7 @@ You would call `signUp` in your application as follows:
 const user = await xprisma.user.signUp('john@prisma.io')
 ```
 
-<Admonition type = "info">
-
-When you call a method in your extension, you must use the correct instance in your code. In the above example, `xprisma.user.signUp` works, but `prisma.user.signUp` does not.
-
-</Admonition>
+When you call a method in an extension, use the constant name from your`$extends`statement, not `prisma`. Instead, use the constant name from your`$extends`statement. In the above example,`xprisma.user.signUp` works, but `prisma.user.signUp` does not.
```

Comment:
Much better, but grammar got a bit weird. I think half sentence is repeated.

Also many missing spaces around the backticks.

Reply:
Good catch - I'll fix in each instance